supplements/van-richtens-guide-to-ravenloft/lycanthrope-wereraven.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+
+	<info>
+        <name>Wereraven Lycanthrope Option</name>
+        <update version="0.0.1">
+            <file name="lycanthrope-wereraven.xml" url="https://raw.githubusercontent.com/mamawerecat/elements/for-pull/supplements/van-richtens-guide-to-ravenloft/lycanthrope-wereraven.xml" />

```

Comment:
Link to wrong repo and needs to also be added to the VRGTR index

Reply:
Done, do i need to submit a new pull reques?

Reply:
no, all changes that you make in the branch apply to the pull